docs/reference/schema.md

```diff
@@ -190,40 +227,50 @@ Schema attributes for an earthquake hazard map related to an occurrence probabil
 |      \*      | Hazard process          | Ground motion |
 |      \*      | Unit of measure         | PGA (g)       |
 
+## Exposure metadata
 
-````
-
-`````
+The `exposure` component is described as:
 
-## Exposure
+```{jsoninclude-quote} ../../docs/_readthedocs/html/rdls_schema.json
+---
+jsonpointer: /properties/exposure/description
+---
+```
 
 The exposure component describes metadata for datasets containing information on the distribution and characteristics of built environment assets (buildings and infrastructure) and natural assets and population, that are used in risk assessment. The exposure component provides codelists to describe the type of assets and costs, and the taxonomy scheme that is used to describe construction and demographic information contained in the dataset. For more information, see [exposure standards](../rdl/other-standards.md#exposure-standards).
 
 The exposure component uses exposure categories consistent with the vulnerability and loss components of this standard. Spatial reference and location information are described using existing external standards. Temporal information can include date and duration of events or year of scenario, and is defined using the Dublin Core standards.
 
-```{eval-rst}
- .. mermaid::
-
-  classDiagram
-      Model -- Asset1
-      Model -- Asset2
-      Model: Category
-      Model: Occupancy
-      class Asset1{
-        Taxonomy code
-        Value type
-        Value unit
-      }
-      class Asset2{
-        Taxonomy code
-        Value type
-        Value unit
-      }
-```
-
-`````{tab-set}
+The following diagram shows a subset of exposure component fields, with required fields highlighted in blue:
 
-````{tab-item} Schema
+```{eval-rst}
+.. uml::
+
+  @startjson
+  <style>
+    .required {
+      BackGroundColor #239ce8
+    }
+  </style>
+  #highlight "exposure" / "category" <<required>>
+  #highlight "exposure" / "metrics" / 0 / "quantity_kind" <<required>>
+  {
+    "exposure": {
+      "category": "",
+      "taxonomy": "",
+      "metrics": [
+        {
+          "dimension": "",
+          "quantity_kind": ""
+        }
+      ]
+    }
+  }
+  @endjson
+
+```
+
+The following table lists all exposure component fields:
```

Comment:
Table still lists metrics as asset cost

Reply:
The tables are generated from the schema, which hadn't changed at the time you reviewed because https://github.com/GFDRR/rdl-standard/pull/204 wasn't yet merged. It is now and I've merged `dev` into this branch so the table has updated auto-magically :-) https://rdl-standard.readthedocs.io/en/178-diagrams/reference/schema/#exposure-metadata